site/styles/_carousel.scss

```diff
@@ -97,15 +97,24 @@ s72-carousel {
   @include media-breakpoint-up(lg) {
     flex-direction: row;
     padding: var(--carousel-bottom-gradient-height) 50px 1px;
-  }
 
-  .sponsor {
-    margin-bottom: 8px;
-    @include media-breakpoint-up(lg) {
+    .sponsor-awards-wrapper {
       align-self: flex-end;
       margin-left: auto;
       order: 1;
     }
+
+    .sponsor {
+      text-align: right;
+    }
+
+    .meta-awards {
+      justify-content: end;
+    }
+  }
+
+  .sponsor {
```

Comment:
Can we have just one `.sponsor` selector in this file please!

Reply:
- changed to one `.sponsor{}` with lg breakpoint inside